status-im/status-react
# Community channel editing

fixes #12065

### Summary

- channel details page added
- channel description can be added on creation 
- channel name and description can be edited

[Related status-go PR](https://github.com/status-im/status-go/pull/2236)

#### Platforms
- Android
- iOS

##### Functional

Community channels editing

### Steps to test
- Open Status
- Create a community or open one where you are an admin
- Create a new channel and make sure you can enter description
- Open channel profile, select edit button (top right) and change name/description
- Make sure changes are seen in a channels list
- 
status: ready
